### ISSUE TITLE
Mailgun mailer sending to cc or bcc without to

### ISSUE THREAD

**Issue by eldair** [symfony/symfony#45548]
### Symfony version(s) affected

6.0.3

### Description

When sending email via mailgun-mailer and setting only `bcc` or `cc` (without setting `to`) exception is thrown `Symfony\Component\Mailer\Exception\HttpTransportException: Unable to send an email: to parameter is missing (code 400).`

### How to reproduce

Try to send email via mailgun and do not set `to` address - use only cc and/or bcc.

**Comment by htmlshaman1** [comment]
It works for me. Using Symfony 6.0.4, tried sending two emails:
only BCC set
only CC set
"To" was not set.

Both emails made it to my inbox. I was using the API, did not use the SMTP transport.

status: works for me

I was using mailgun, and the mailer component.

**Comment by eldair** [comment]
Hm, this is in laravel 9, maybe the issue is on their side then. I will investigate further.

**Comment by eldair** [comment]
@htmlshaman1 I recreated code using only symfony components and it still throws the same exception.

```
$transpor = new MailgunApiTransport(env('MAILGUN_SECRET'), env('MAILGUN_DOMAIN'));
$mailer = new Mailer($transpor);

$email = (new Email())
    ->from('test@example.com')
    ->bcc('test1@example.com', 'test2@example.com')
    ->subject('Time for Symfony Mailer!')
    ->text('Sending emails is fun again!')
    ->html('<p>See Twig integration for better HTML integration!</p>');

$mailer->send($email);
```

How did you set `bcc` without `to`?

**Comment by eldair** [comment]
Correction, it works with `MailgunHttpTransport`

**Comment by andreladocruz** [comment]
I'm getting the same error here now... =( 

Symfony\Component\Mailer\Exception\HttpTransportException: Unable to send an email: to parameter is missing (code 400). in /var/www/html/vendor/symfony/mailgun-mailer/Transport/MailgunHttpTransport.php:81

Any clue?

**Comment by OzanKurt** [comment]
It also suddenly appeared by me? Any ideas? @andreladocruz

**Comment by andreladocruz** [comment]
@OzanKurt,

I found that it was related to the latest php version for alpine:edge container image.

In container built on Mar, 13th (php 8.3.3) it works.

In container built on Apr, 1st (php 8.3.4) it fails.

This is very strange. =P

**Comment by OzanKurt** [comment]
@andreladocruz 

I'm in PHP 8.2.7 at the moment and my local is 8.2.0

Oh god this is insane!

**Comment by andreladocruz** [comment]
@OzanKurt,

Very insane!!!! =) =)

**Comment by ksvendsen** [comment]
Today i am having same problem in production, but not on local.

By the way, not using BCC or CC only using TO and same error

**Comment by OzanKurt** [comment]
Exactly the same thing is happening to me

**Comment by paulheijmans** [comment]
Same issue here. Is there already a solution for this?

**Comment by AlejandroMuralles** [comment]
I´m having the same problem. Any ideas?

`Unable to send an email: to parameter is missing (code 400).`

**Comment by OzanKurt** [comment]
I ended up migrating from mailgun to mailersend. It was an extremely stressful day.

**Comment by xabbuh** [comment]
this is probably the same as the current issue #54491, someone with Mailgun credentials will need to debug what happened that the communication with Mailgun suddenly fails

**Comment by RoliJ** [comment]
Same issue here. Still no luck?

**Comment by mfedak-tqgg** [comment]
Looks like the problem came from a PHP version update.

It started happening to me after the hosting provider updated the php version to 8.1.27 2 days ago.

With my local version PHP 8.3.3 I do not have any issues.

@michaelhue suggested fix which is working for me: https://github.com/symfony/symfony/issues/54491#issuecomment-2040637747

**Comment by nickfls** [comment]
Happens with:

PHP: 8.2.17
Laravel: 9.52.16
Symfony: mailer: v6.4.6 | mailgun-mailer: v6.4.4

**Comment by DamianUS** [comment]
Why is this closed? It happens with 8.1.27.

**Comment by xabbuh** [comment]
see #54491
